main.go

```diff
@@ -106,55 +107,70 @@ func main() {
 		payload.WindowsExporterPath,
 	}
 	if err := checkIfRequiredFilesExist(requiredFiles); err != nil {
-		log.Error(err, "could not start the operator")
+		setupLog.Error(err, "could not start the operator")
 		os.Exit(1)
 	}
 
-	// The watched namespace defined by the WMCO CSV
-	namespace, err := k8sutil.GetWatchNamespace()
+	ctx := context.TODO()
+
+	// Create a new Manager to provide shared dependencies and start components
+	// The NewCache field is not being set as it will default to a cluster wide scope as we need to watch Nodes.
+	// Namespaces should be excluded using predicate functions when defining watches.
+	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+		Scheme:             scheme,
+		MetricsBindAddress: fmt.Sprintf("%s:%d", metrics.Host, metrics.Port),
+		Port:               9443,
+		LeaderElection:     enableLeaderElection,
+		LeaderElectionID:   "86f835c3.example.com",
+	})
 	if err != nil {
-		log.Error(err, "failed to get watch namespace")
+		setupLog.Error(err, "unable to start manager")
 		os.Exit(1)
 	}
 
-	ctx := context.TODO()
-	// Become the leader before proceeding
-	err = leader.Become(ctx, "windows-machine-config-operator-lock")
-	if err != nil {
-		log.Error(err, "failed to become a leader within current namespace")
+	// Setup Scheme for all resources
+	if err := apis.AddToScheme(mgr.GetScheme()); err != nil {
+		setupLog.Error(err, "failed to add all Resources to the Scheme")
 		os.Exit(1)
 	}
 
-	// Allow for the watching of cluster-wide resources with "", so that we can watch nodes,
-	// as well as resources within the `openshift-machine-api` and WMCO namespace
-	namespaces := []string{"", "openshift-machine-api", namespace}
-	// Create a new Cmd to provide shared dependencies and start components
-	mgr, err := manager.New(cfg, manager.Options{
-		NewCache:           cache.MultiNamespacedCacheBuilder(namespaces),
-		MetricsBindAddress: fmt.Sprintf("%s:%d", metrics.Host, metrics.Port),
-	})
+	wmcoNamespace, err := getWatchNamespace()
 	if err != nil {
-		log.Error(err, "failed to create a new Manager")
+		setupLog.Error(err, "failed to get watch namespace")
+		os.Exit(1)
+	}
+	if wmcoNamespace == "" {
+		setupLog.Error(err, "WMCO running with cluster scope")
```

Comment:
It can happen if someone changes the CSV or WATCH_NAMESPACE env var. Added this check to be safe as it will result in the operator not working properly

Reply:
How can they change those values without changing the `Deployment`? Won't OLM stomp on those changes? I am fine with being defensive but there needs to stated reasons for such checks.